### DISCUSSION TITLE
Please help getting this web radio url

### DISCUSSION THREAD

**Discussion by ditusade** [jcorporation/myMPD#1041] [General]
Hi,
I have been able to add different radio streaming url into my playlist in myMPD.
However, I can't do the same for http based streaming radio (e.g link: https://mbcradio.tv/streaming/bestfm.html)
The Network shows m38u and ts files with no direct url.
I thought this can be done by writing a script using mympd.http_client(method, uri, headers, payload) but I dont have the skills to write the code.
Would really appreciate if anyone could help it working please.
Thank you

**Comment by jcorporation** [comment]
You can use "Add stream" from the main menu to add any stream MPD supports. I do not know, If MPD supports ts files.

Scripting can't help you in this case. MPD must support the media type.

**Reply by ditusade** [reply]
Ah ok. Thx for the reply
